sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi.c

```diff
@@ -85,6 +94,14 @@ bool act_is_servo[INDI_NUM_ACT] = STABILIZATION_INDI_ACT_IS_SERVO;
 bool act_is_servo[INDI_NUM_ACT] = {0};
 #endif
 
+#ifdef STABILIZATION_INDI_ACT_PREF
+// Preferred (neutral, least energy) actuator value
+float act_pref[4] = STABILIZATION_INDI_ACT_PREF;
```

Comment:
I guess this should be INDI_NUM_ACT instead of hardcoded. Also on line 130 for mu1